2-mandatory/6-journey-planner.js

```diff
@@ -17,15 +17,12 @@
  Hint: search for string methods like Includes and IndexOf.
 */
 
-function checkCodeIsThere(stringText) {
+const checkCodeIsThere = (stringText) => {
```

Comment:
Perfect again! The only very slight suggestion I have is to try and use `const` over `let` if we are not reassigning the value of the variable 🙌

Reply:
Sure :)